public/index.html

```diff
@@ -64,6 +64,14 @@
     <meta name="twitter:image" content="./assets/img/logos/wildcards.png" />
     <meta name="twitter:card" content="./assets/img/logos/wildcards.png" />
 
+    <!-- 
+      Make the website not scrollable horizontally on mobile.
+      src:https://stackoverflow.com/a/51317940/3103033 -->
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale = 0.86, maximum-scale=3.0, minimum-scale=0.86"
```

Comment:
Ok, interesting, it seems this doesn't work on safari. The stackoverflow question I got this from has some other solutions. Maybe they will work better safari. It had no side scroll on any of the browsers I tested.

Using `overflow: hidden` might help actually, I didn't try it because I wanted to use the same styles as the desktop.

Reply:
Fixed with `overflow: hidden`.